pages/admin/[id].js

```diff
@@ -0,0 +1,37 @@
+import { useRouter } from "next/router";
+import { useState } from "react";
+import { useEffect } from "react";
```

Comment:
```suggestion
import { useState, useEffect } from "react";
```
Shorter and more readable ;-)